sdcm/nemesis.py

```diff
@@ -1056,6 +1056,35 @@ def disrupt_modify_table(self):
         disrupt_func = getattr(self, disrupt_func_name)
         disrupt_func()
 
+    def disrupt_mgmt_backup(self):
+        self._set_current_disruption('ManagementBackup')
+        if not self.cluster.params.get('use_mgmt', default=None):
+            raise UnsupportedNemesis('Scylla-manager configuration is not defined!')
+        if not self.cluster.params.get('backup_bucket_location'):
+            raise UnsupportedNemesis('backup bucket location configuration is not defined!')
+
+        manager_node = self.monitoring_set.nodes[0]
+        manager_tool = mgmt.get_scylla_manager_tool(manager_node=manager_node)
+        self.log.debug("sctool version is : {}".format(manager_tool.version))
+
+        cluster_name = self.cluster.name
+        mgr_cluster = manager_tool.get_cluster(cluster_name)
+        if not mgr_cluster:
+            self.log.debug("Could not find cluster : {} on Manager. Adding it to Manager".format(cluster_name))
+            targets = [[n, n.ip_address] for n in self.cluster.nodes]
+            mgr_cluster = manager_tool.add_cluster(name=cluster_name, host=targets[0][1], disable_automatic_repair=True,
+                                                   auth_token=self.monitoring_set.mgmt_auth_token)
+        bucket_location_name = self.cluster.params.get('backup_bucket_location').split()
+        region = self.cluster.params.get('region_name').split()
+        for node in self.cluster.nodes:
+            update_config_file(node=node, region=region[0])
+        mgr_task = mgr_cluster.create_backup_task({'location': ['s3:{}'.format(bucket_location_name[0])]})
+        mgr_task.wait_for_status([TaskStatus.DONE], timeout=43200)  # timeout is 12 hours
```

Comment:
What if the status is not DONE?
For example FAILED, or even stuck on NEW?
Will it wait the entire timeout?

If so, that's need to be changed.

Reply:
the thing with FAILED status, is that it can "change" since it automatically has 4 "tries" before final result... we could indeed check if it failed for the 4th time, and exit, instead of wait for timeout... but we don't have a good way to check if it is stuck in NEW status... I would expect the manager to fail it after some time, so it will be failed, not stay new forever.

Reply:
Please fix, it shouldn't wait for the whole timeout in any case but very long backup.
If the status changed to anything but running, fail the nemesis.

Reply:
added a trello task to track improvements in the current code

Reply:
Did you send a PR for the things to be fixed here?
If not, please send ASAP.